### DISCUSSION TITLE
Is it possible to log response body of 4xx responses ?

### DISCUSSION THREAD

**Discussion by tizba** [zalando/logbook#1575] [Q&A]
Hi,

Title basically.
Is it possible to log the response bodies of 4xx responses using the Spring Boot starter ?
It seems that it, by default, only logs the response bodies of 2xx responses.

Thanks in advance 😄
Baptiste

**Comment by whiskeysierra** [comment]
- Describes which 4xx responses are logged without body and why: https://github.com/zalando/logbook#security
- Describes how to conditionally log the body: https://github.com/zalando/logbook#strategy
- Describes how to configure ^ based on spring application properties: https://github.com/zalando/logbook#configuration

**Reply by tizba** [reply]
Seems to me that by default Logbook does not log authorized 400 or 404 responses and that [https://github.com/zalando/logbook#security](https://github.com/zalando/logbook#security) does not describes that

**Reply by whiskeysierra** [reply]
400 and 404 can't be explained with the special security handling, that's correct.
Can you create a minimal example project to reproduce it?

**Reply by tizba** [reply]
yes for sure : https://github.com/tizba/logbook-not-logging-response-body
